src/types/configuration/CryptoConfiguration.ts

```diff
@@ -1,4 +1,7 @@
 export default interface CryptoConfiguration {
   key: string;
   algorithm: string;
+  blockCypher?: string;
+  keySize?: number;
+  operationMode?: string;
```

Comment:
These 3 fields are key dependent settings. They need to be attached to a key, not the global configuration.